react-client/src/components/PickMissionPlayers.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+class PickMissonPlayer extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+  }
+
+  render() {
+
+    return (
+      <div>
+
+      <Stats />
+
+      <Timer />
+
+        <h3> Pick Mission Players </h3>
+
+        Pick {this.props.missionCount} players to go:
```

Comment:
maybe instead of 'pick' we should have 'enter' because pick implies that user gets to make the decision, rather than just enter the agreed upon mission participants